packages/core/integration-tests/test/monorepos.js

```diff
@@ -856,7 +876,7 @@ describe('monorepos', function () {
       ),
       'utf8',
     );
-    assert(contents.includes('import "./pkg-b.cjs.css"'));
+    assert(contents.includes('import "./pkg-b.module.css"'));
   });
 
```

Comment:
changes in this file are due to css bundles no longer being reused between different targets, so there is now both a CJS and ESM version